### PR TITLE
feat:Added terms conditions and privacy policy

### PR DESCRIPTION
**Pull request by pranalidhanavade**
## What 

- Added terms and conditions 
- Added privacy policy

### CODE REVIEW

File: src/styles/markdown.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+body {
+    background-color: #ffffff;
+    font-family: ui-sans-serif, system-ui, "Segoe UI", "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji", ui-sans-serif, system-ui, system-ui, Helvetica ,Arial, sans-serif,Color, Symbol;
+    ;
+    /* margin-left: 10rem;
+    margin-right: 10rem; */
```
Comment (nishad-ayanworks) [inline src/styles/markdown.css:6]: Please remove the commented code.

**Review by nishad-ayanworks — CHANGES_REQUESTED** [review]
Please check comments

File: src/styles/markdown.css (hunk 1 above)
Comment (pranalidhanavade) [reply src/styles/markdown.css]: Resolved

**Review by nishad-ayanworks — APPROVED** [review]
LG

**Pull request merged**